### PR TITLE
[MIG] website_hr_department migrate to v10

### PR DESCRIPTION
**Pull request by JonathanNEMRY**

### CODE REVIEW

**Comment by JonathanNEMRY** [comment]
Coverage make the PR red but it is actually green and ready to review :smile:

File: website_hr_department/views/website_hr_department.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
               <div class="col-sm-5" t-if="department">
                 <t t-call="website.publish_management">
                   <t t-set="object" t-value="department"/>
+                  <t t-set="publish_edit" t-value="True"/>
```
Comment (Tardo) [inline website_hr_department/views/website_hr_department.xml:22]: When use "publish_edit" doesn't need define "action" too?

**Review by Tardo — COMMENTED** [review]
And other stuff not related with this module... Can be good feature extend in "publish_management" template an option for hide the dropdown button.. no?

**Comment by simahawk** [comment]
@JonathanNEMRY still working on this?

**Review by simahawk — APPROVED** [review]

**Pull request closed without merging**